CONTRIBUTING.md

```diff
@@ -0,0 +1,6 @@
+Contributions to ASiMoV-CCS are welcomed via pull requests.
+
+When preparing a contribution we ask that you read the developers guide (see `dev_guide/`) to check for design and coding style intentions.
+As described in the build tools documentation (`build_tools/build_system_readme.md`) the `flint` program is used to apply linting rules to the ASiMoV-CCS source.
+
+Before a contribution is merged it will be reviewed by one of the ASiMoV-CCS developers, in particular the full test suite must still pass.
```

Comment:
I would change the final sentence slightly:

"Before a contribution can be merged, it will be reviewed by one of the ASiMoV-CCS developers, in particular to check the coding standard is followed and to verify the full test suite must still pass."